app/src/test/java/ar/scacchipa/twittercloneapp/repository/AuthorizationRepositoryTest.kt

```diff
@@ -46,6 +48,47 @@ class AuthorizationRepositoryTest {
 
         Assert.assertEquals(expected, actual)
     }
+
+    @Test
+    fun whenAuthRepoReceiveRequestWithoutBodyReturnError() = runTest {
```

Comment:
Seems ok but here we use mockito to model de mock behavior , which is ok, but in the next test we create the mock by hand, maybe we could go for mockito in all the test.